test/integration/webhook/jobset_webhook_test.go

```diff
@@ -155,6 +168,40 @@ var _ = ginkgo.Describe("jobset webhook defaulting", func() {
 				return js.Spec.ReplicatedJobs[0].Template.Spec.Template.Spec.RestartPolicy == corev1.RestartPolicyOnFailure
 			},
 		}),
+		ginkgo.Entry("validate jobSet should fail on NodeSelectors Update", &testCase{
```

Comment:
we need a test that creates a suspended jobset and the update succeeds.

Reply:
Should I remove "validate jobSet immutable for fields over than NodeSelector when jobSet is not suspended" test from line 202 in this case?

Reply:
no, that should stay because it validates a different case. The test case I am asking for is exactly like this one, but the jobset is created suspended and so `updateShouldFail=false`

Reply:
sg, thanks